epub33/epub-a11y-eaa-mapping/index.html

```diff
@@ -58,8 +58,7 @@
 				disabilities to access goods and services, including ebooks, dedicated reading software, ereading
 				devices, and ecommerce.</p>
 
-			<p>The EAA has been approved on 27 June 2019, should be implemented by EU Member States within 28 June 2022
-				and will enter in force from 28 June 2025.</p>
+			<p>The EAA has been approved on 27 June 2019 and will come into force from 28 June 2025. The directive has already been implemented at national level in many member states and is being implemented in the remaining.</p>
```

Comment:
```suggestion
			<p>The EAA was approved on 27 June 2019 and will come into force from 28 June 2025. Many member states have already implemented the directive at the national level, with the remaining states currently in the process of implementing it.</p>
```

"has been" suggests a continuous state from the past (e.g., "has been approved since..." but I don't think that's what you want here). The second sentence doesn't need to be passive.